src/StatusQ/Core/Theme/ThemePalette.qml

```diff
@@ -90,7 +90,20 @@ QtObject {
     property color miscColor11
     property color miscColor12
 
-    property var userCustomizationColors: []
+    property var userCustomizationColors: [
+        getColor('black', 0.8),
+        getColor('grey'),
+        getColor('blue2'),
+        getColor('purple'),
+        getColor('cyan'),
+        getColor('violet'),
+        getColor('red2'),
+        getColor('yellow'),
+        getColor('green2'),
+        getColor('moss'),
+        getColor('brown'),
+        getColor('brown2'),
+    ]
```

Comment:
With this change, both themes get the same set of colors which was not the case before. Are we sure we want this?

Reply:
Please note that User Customisation colors have the same name and yet are slightly different. Ideally, we should use color indexes, as we use for user colors now.

Reply:
Currently, for user icon colors, we use a color index, see: https://github.com/status-im/status-desktop/blob/19235e2b7940a1c686693106a4533ca47c9f8d23/ui/imports/utils/Utils.qml#L665-L674

> But as I checked, we store the color, not it's index.

The color index for user icons is deterministically obtained from pubKey and so there is no need to store it in the database.

The unencrypted accounts database (the one responsible for keeping information about accounts you can log in to) contains a color index though. That's because pubKey is not accessible before we log in.

In other scenarios that's something we should either fix (migrate color to color index) in order to solve the problem you mentioned:
> As we never save the "index of unversal color" but the color value itself, then it's possible that the color was selected using Dark theme and will be viewed later in Light theme.

Or make user customization colors stay the same in every theme.

We should not solve it by simply preferring fixed colors from outdated Designs, IMO we should not merge this change.